docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+.. |Dockstore Logo Zip| image:: /assets/images/Dockstore_Logos_and_Icons.zip
+
+.. |Dockstore Dot| image:: /assets/images/Dockstore-blue.png
+  :width: 64
+
+.. |Vertical Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-blue.png
+  :width: 128
+
+.. |Vertical White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-white.png
+  :width: 128
+
+.. |Vertical Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black.png
+  :width: 128
+
+.. |Horizontal Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-blue.png
+  :width: 256
+
+.. |Horizontal White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white.png
+  :width: 256
+
+.. |Horizontal Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black.png
+  :width: 256
+
+.. |Vertical White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-white-blue.png
+  :width: 128
+
+.. |Vertical Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black-blue.png
+  :width: 128
+
+.. |Horizontal White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white-blue.png
+  :width: 256
+
+.. |Horizontal Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black-blue.png
+  :width: 256
+
+.. |Icon Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-blue.png
+  :width: 64
+
+.. |Icon Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-black.png
+  :width: 64
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
```

Comment:
I'd replace this heading with "Background"

Reply:
The SVG files are in the zip file along with the PNG files.

Why 800x600?   That seems large to me for most use.  I can scale them up, but, then what is the magic size.

The horizontal ones are 406 x 97.
The vertical ones are 253 x 147.
The icons are 140 x 97.

I do want to note that the images are scaled down in the table to get them to fit comfortably in the table format within the fixed width window.